environments/dap/base/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable project_name {
+  type = string
+}
+
+variable vault_prefix {
+  type = string
```

Comment:
can we include descriptions for variables like this? i'd want to know things like "what the prefix should start with" and "what will be appended to it" for a `vault_prefix` variable